RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeeds_cfi.py

```diff
@@ -34,11 +34,18 @@
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
     ecalDrivenElectronSeeds,
-    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal')
+    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal'),
+    SeedConfiguration = dict( allowHGCal = cms.bool(True) )
 )
+
+
+# create ecal driven seeds for electron using HGCal Multiclusters
+ecalDrivenElectronSeedsFromMultiCl = ecalDrivenElectronSeeds.clone()
```

Comment:
cfi files should not be used for definition of multiple modules.
Please move to ecalDrivenElectronSeedsModules_cff

technically ```phase2_hgcal.toModify( ``` is not needed for ecalDrivenElectronSeedsFromMultiCl and the necessary change to endcapSuperClusters can be passed with the clone arguments
```
ecalDrivenElectronSeedsFromMultiCl = ecalDrivenElectronSeeds.clone(endcapSuperClusters = 'particleFlowSuperClusterHGCalFromMultiCl') 
```